### DISCUSSION TITLE
Changing stacked Toast order

### DISCUSSION THREAD

**Discussion by ollie-bud** [radix-ui/primitives#2097] [Help] [answered]
Hi there - absolutely loving Radix 🙌 

I've been using the toast component and I wondered if stacked toasts can be reordered (so that new toasts come in at the top, rather than the bottom) a little like https://apvarun.github.io/toastify-js/#

Thanks,
Ollie!

**Comment by joaom00** [comment, marked as answer]
something like this? https://codesandbox.io/s/toast-imperative-notifications-8jfl32

**Reply by ollie-bud** [reply]
Yes! That's absolutely perfect! Thanks so much <3
